### PR TITLE
Add more Free Code Camp places in README.md

### PR DESCRIPTION
**Pull request by IsaacAbrahamson**
#### Proposed documentation enhancement to README.md

Free Code Camp also has an awesome forum and wiki which are not mentioned with our other places (Gitter, Youtube, Medium, etc.). I propose we add links to these places in the relevant section as people viewing the repository may be interested in viewing them.

Issue: https://github.com/freeCodeCamp/freeCodeCamp/issues/12502

Old pull request: https://github.com/freeCodeCamp/freeCodeCamp/pull/12503

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,7 +43,7 @@ For this certification, you'll work on **two projects from scratch** and then **
 
 ---
 
-This code is running live at [FreeCodeCamp.com](https://www.freecodecamp.com). We also have [Gitter chat rooms](https://gitter.im/FreeCodeCamp/FreeCodeCamp), a [Medium publication](https://medium.freecodecamp.com), and even a [YouTube channel](https://youtube.com/freecodecamp).
+This code is running live at [FreeCodeCamp.com](https://www.freecodecamp.com). We also have [Gitter chat rooms](https://gitter.im/FreeCodeCamp/FreeCodeCamp), a [Medium publication](https://medium.freecodecamp.com), an [interactive forum](https://forum.freecodecamp.com), a [technology wiki](https://forum.freecodecamp.com/c/wiki), and even a [YouTube channel](https://youtube.com/freecodecamp).
```
Comment (erictleung) [inline README.md]: Maybe instead of "technology wiki", we could make it "Wiki knowledgebase"?

Comment (IsaacAbrahamson) [inline README.md]: Agreed, that sounds better. Should wiki be capitalized? It is not a brand, proper noun, etc.

Comment (IsaacAbrahamson) [inline README.md]: Fixed.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
#### :sparkles: :sparkles: **LGTM.** :sparkles: :sparkles: 

Thanks @IsaacAbrahamson Awesome work, and congrats on your first PR to freeCodeCamp.

FYI: Checkout our [Guidelines for Contributing](https://github.com/FreeCodeCamp/FreeCodeCamp/blob/staging/CONTRIBUTING.md), on how to name branches, keep your local fork in sync, and rebasing.

**Comment by IsaacAbrahamson** [comment]
@raisedadead Awesome, thanks so much. Now I feel like I have actually done something!

**Pull request merged**